elasticsearch_dsl/search.py

```diff
@@ -494,6 +500,60 @@ def script_fields(self, **kwargs):
         s._script_fields.update(kwargs)
         return s
 
+    def knn(
+        self,
+        field,
+        k,
+        num_candidates,
+        query_vector=None,
+        query_vector_builder=None,
+        filter=None,
+        similarity=None,
```

Comment:
Looks like `boost` is missing. It's specified here: https://github.com/elastic/elasticsearch-specification/blob/05c39a9dc46e107e4baaf082a6b56aae1ef5901c/specification/_types/Knn.ts#L26-L43. Documented here: https://www.elastic.co/guide/en/elasticsearch/reference/current/knn-search.html#_search_multiple_knn_fields. But weirdly missing fom https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html#search-api-knn.

Reply:
@pquentin I wonder why is `boost` omitted from our documentation. But in any case, I have added it now.